integration-cli/docker_cli_cp_test.go

```diff
@@ -528,3 +529,30 @@ func TestCpToDot(t *testing.T) {
 	}
 	logDone("cp - to dot path")
 }
+
+func TestCpToStdout(t *testing.T) {
+	out, exitCode, err := dockerCmd(t, "run", "-d", "busybox", "/bin/sh", "-c", "echo lololol > /test")
+	if err != nil || exitCode != 0 {
+		t.Fatalf("failed to create a container:%s\n%s", out, err)
+	}
+
+	cID := stripTrailingCharacters(out)
+	defer deleteContainer(cID)
+
+	out, _, err = dockerCmd(t, "wait", cID)
+	if err != nil || stripTrailingCharacters(out) != "0" {
+		t.Fatalf("failed to set up container:%s\n%s", out, err)
+	}
+
+	out, _, err = runCommandPipelineWithOutput(
+		exec.Command(dockerBinary, "cp", cID+":/test", "-"),
+		exec.Command("tar", "-vtf", "-"))
+	if err != nil {
+		t.Fatalf("Failed to run cp: %s", err)
```

Comment:
Ah, I didn't realize that's what you meant.  Sorry.  Next PR, if I remember, I'll sneak it in  :-)